.github/workflows/ubuntu-ci.yml

```
@@ -1,68 +1,58 @@
-  # deprecated actions/download-artifact@v3 + actions/create-release@v1; those
-  # now execute only on the rare tag push and would need replacing to actually
-  # cut a release.
+  # Runs only on tag pushes. NOTE: still uses the deprecated
+  # actions/create-release@v1 + download-artifact@v3; they would need replacing
+  # to actually cut a release, but only fire on the rare tag push.
   create-release:
     needs: build-and-package
     runs-on: ubuntu-latest
-    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')
     steps:
     - uses: actions/download-artifact@v3
```

actionlint (1.7.12)

[error] the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)